Documentation/User Groups.md

```diff
@@ -0,0 +1,15 @@
+# User Groups
+Listed are the main user groups for Home Together:
+- People in Canada, mainly aged 50 and up, looking for people to homeshare with or homesharing services.
+- Homesharing businesses looking to advertise their homesharing services
+
+1. Users who are looking for housing
+2. Users who are looking for housemate(s) (non-profit)
+3. Users who are looking for tenant(s) (for Profit) 
+4. Users who are looking for services
+5. Users who are selling services
+
+There may be overlap between groups:
+Eg. User x may be looking for a roommate (2) and to hiring house cleaning services (4).
```

Comment:
Formatting should be standardized, and possibly split into sections. The second list here is a more detailed one, perhaps better as a sublist?

Also, I'm not sure if point 3. here fits the client's description of Home Sharing. It's not meant to be the same as renting out a house for profit; it's a mutual thing where the house is shared equally between members.

Reply:
I was under  the impression by " If you wish to post a classified ad, payment goes through a secure on-line service.  (example pay pal)" in FAQs Final Oct 22nd that users would be able to place classified ads which to me presumed profit.

Reply:
The ads part of the project has been in ongoing discussion with the client. At this point we don't have a great answer about how these should be integrated into the website. However, I am fairly confident that the ads are *not* classified ads, but rather an ad space for local businesses on the website. This is a primary discussion point for Thursday's meeting; see the meeting notes on Discord.

Reply:
I'm going to split this off into it's own issue, we can finalize it after the meeting with the client when we have more clarification.

Reply:
Sounds good.